fem/bilininteg_dgtrace_pa.cpp

```diff
@@ -167,6 +167,16 @@ void DGTraceIntegrator::SetupPA(const FiniteElementSpace &fes, FaceType type)
       r.SetSize(1);
       r(0) = c_rho->constant;
    }
+   else if (QuadratureFunctionCoefficient* c_rho =
+               dynamic_cast<QuadratureFunctionCoefficient*>(rho))
+   {
+      //Assumed to be in lexicographical ordering
```

Comment:
Does this comment really apply here since the QF here only has a VDIM of 1? I can see it applying to the ```fem/bilininteg_convection_pa.cpp``` addition though.

Reply:
Good point!